packages/core/theme-module/ensureDirectoryExists.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
```

Comment:
maybe lets put all those files into a `scripts` folder. It slowly becomes messy

Reply:
fixed